/* Copyright 2023 The FedLearner Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

package pp_lite.deploy.configs;

message Config {
    // Prefix of the certificate: <pure_domain_name>.fedlearner.net
    string pure_domain_name = 1;
    string image_uri = 2;
    // Whether to include pp_lite_client image in the produced zip file
    bool include_image = 3;
    string auto_cert_api_key = 4;
}
